cabby/geo/visualization/templates/follower_task.html

```diff
@@ -0,0 +1,146 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<br>
+
+
+<div class="float-container">
+    <div class="float-child">
+        <h3>Navigation Map</h3>
+
+        <b>Map Instructions: </b>
+
+        <ul>
+            <li>You can zoom in and out by clicking on the +/- button.</li>
+            <li>You can press and hold to move on the map.</li>
+
+        </ul>
+
+
+        
+<div id="mapid" class=map></div>
+<script>
+
+    var map = L.map('mapid').setView({{start_point|tojson}}, 16);
+
+
+	L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw', {

```

Comment:
Is this a private access token that shouldn't be checked into to OS code? (If so, need to regenerate and pass in another way.)